Scarlet/README.txt

```diff
@@ -1,2 +0,0 @@
-Compiled commit f02b1ffda6e39f6e9ac6969b4bdce23e3e759581, downloaded from https://github.com/TheCrazyT/Scarlet/

```

Comment:
I don't really want to delete this file, it contains important information in my opinion.

Reply:
I've added the SCARLET Readme file back, I had forgotten to check how much of scarlet was part of this repository.